.travis.yml

```diff
@@ -24,10 +24,10 @@ cache:
 
 before_install: . ./ci/before_install.sh
 
+before_script: cd libbindgen
+
 script:
-  - cargo build --verbose --features "$BINDGEN_FEATURES"
   - cargo test --features "$BINDGEN_FEATURES"
-  - cargo build --release --verbose --features "$BINDGEN_FEATURES"
   - cargo test --release --features "$BINDGEN_FEATURES"
```

Comment:
One more +1 to building the binary in CI: we need to ensuring that master always has a compiling binary is a very minimal standard we need to hold ourselves to.